client/components/nav-bar.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import {Collapse, Navbar, NavbarToggler, NavbarBrand, Nav, NavItem,NavLink } from 'reactstrap'
+
+export default class NavBar extends Component {
+  constructor(props) {
+    super(props);
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      isOpen: false
+    };
+  }
+  toggle() {
+    this.setState({
+      isOpen: !this.state.isOpen
+    });
+  }
+  render() {
+    return (
+      <div>
+        <Navbar className="container fixed-top" color="white" light expand="md">
+          <NavbarBrand className="ml-5" id="brand" href="/">Volab</NavbarBrand>
+          <NavbarToggler onClick={this.toggle} />
+          <Collapse isOpen={this.state.isOpen} navbar>
+            <Nav className="ml-auto mr-5" navbar>
+              <NavItem>
+                <NavLink href="https://github.com/franzsalvador">Blog</NavLink>
+              </NavItem>
+              <NavItem>
+                <NavLink>Help</NavLink>
```

Comment:
Don't include links or buttons that don't do anything yet. It makes your project appear "broken". `master` should only ever include functioning features.